graph/src/components/subgraph/proof_of_indexing/reference.rs

```diff
@@ -9,17 +9,20 @@ use web3::types::{Address, H256};
 /// well-implemented (without conflicting sequence numbers, or other oddities).
 /// It's just way easier to check that this works, and serves as a kind of
 /// documentation as a side-benefit.
+#[allow(dead_code)]
```

Comment:
This should be moved into the `tests` module in  `graph/src/components/subgraph/proof_of_indexing/mod.rs`.